helm/values/env-bleed.yaml

```diff
@@ -0,0 +1,48 @@
+# Values and overrides for the bleed env for the cms-apps chart
+
+global:
+  hostname: "wagtail-poc.bleed.onsdigital.co.uk"
+
+wagtail:
+
+  ingress:
+    hostName: "wagtail-poc.bleed.onsdigital.co.uk"
+
+  serviceAccount:
+    iam: "arn:aws:iam::697746439640:role/wagtail-poc"
+
+  extraEnv: |-
+    - name: ALLOWED_HOSTS
+      value: "*"
+    - name: BIRDBATH_REQUIRED
+      value: "False"
+    - name: PRIMARY_HOST
+      value: "{{ .Values.global.hostname }}"
+    - name: WAGTAILADMIN_BASE_URL
+      value: "https://{{ .Values.global.hostname }}"
+    - name: PG_DB_ADDR
+      value: "wagtail-content-pg-rds.cluster-cyuiulg5fce0.eu-west-2.rds.amazonaws.com"
+    - name: PG_DB_PORT
+      value: "5432"
+    - name: ELASTICACHE_ADDR
+      value: "wagtail-poc-redis-serverless-rjbupb.serverless.euw2.cache.amazonaws.com"
+    - name: ELASTICACHE_PORT
+      value: "6379"
+    - name: AWS_STORAGE_BUCKET_NAME
+      value: "wagtail-ons-poc-media"
+
+  secrets:
+    - secret: django-secret-key
+      data:
+        secret-key: dnhrJm5ndmdoMzF1d21eN2opK3k5NUB1X25yITB1ZWhwMnNkPTAkeSFAZnA5Mm9saGxeYjVuLXRu
```

Comment:
Note: Currently, this value is exposed in the repository. It's currently not used for anything, as the entire environment is protected by IP restriction.

Proper secret management is in progress.